src/main/knn/KnnGraphTester.java

```diff
@@ -557,7 +565,22 @@ private void testSearch(Path indexPath, Path queryPath, Path outputPath, int[][]
       totalVisited /= numIters;
       System.out.printf(
           Locale.ROOT,
-          "%5.3f\t%5.2f\t%d\t%d\t%d\t%d\t%d\t%d\t%.2f\t%s\n",
+          "|%s\t|%s\t|%s\t|%s\t|%s\t|%s\t|%s\t|%s\t|%s\t|%s|\n",
```

Comment:
Sounds right. Will revert this part. My thought was that the output is generally pasted in the comments here so it would be better to pre-format it into md table.

Reply:
I have reverted to csv format.